data-pipeline/src/trace_exporter.rs

```diff
@@ -108,6 +118,11 @@ impl<'a> From<&'a TracerTags> for HashMap<&'static str, String> {
     }
 }
 
+enum HealthMetric {
+    Count(&'static str, i64),
+    //TODO: Add more DogStatsDAction as we need them
```

Comment:
nit: This TODO comment seems unnecessary? When the time is right to add to the enum with new DogStatsDActions someone will just do it. There is no tech-debt here to address later.